test/index.js

```diff
@@ -4,18 +4,11 @@
 require('source-map-support').install();
 
 global.sinon = require('sinon');
+require('./chai-setup');
+global.chai.use(require('sinon-chai'));
 
-const chai = (global.chai = require('chai'));
-
-chai.use(require('sinon-chai'));
-chai.should();
-
-const bluebird = require('bluebird');
-global.expect = chai.expect;
 global.d = new Date();
 
-bluebird.longStackTraces();
-
```

Comment:
One concern: `bluebird.longStackTraces()` has actually been very helpful in tracking down race conditions in the code!  We might want to figure out a way to optionally enable this feature just for testing purposes.

(The challenge, of course, is that it then also becomes difficult to verify that the code is working properly _without_ relying upon `Bluebird`.  It's like it needs to be a test dependency, but also _not_ be a test dependency at the same time)

cc: @kibertoad

Reply:
Not affects tests directly, so we can just return it back and remove in second pr